src/lib/logger-types.ts

```diff
@@ -94,5 +94,12 @@ export enum LogEventName {
 
   LOADING_MEASUREMENTS,
 
-  EXEMPLAR_VISIBILITY_UPDATE
+  EXEMPLAR_VISIBILITY_UPDATE,
+
+  TOOLBAR_EDIT_TOOL,
+  TOOLBAR_SELECT_ALL_TOOL,
+  TOOLBAR_COPY_TO_DOCUMENT,
+  TOOLBAR_COPY_TO_WORKSPACE,
+  TOOLBAR_PLAYBACK_TOOL,
+  TOOLBAR_FOUR_UP_TOOL,
```

Comment:
Have you checked what was previously logged for the buttons that were just moved - edit, 4up, playback - for continuity in case research spans across this release?

Reply:
I checked and nothing was logged for edit or playback.  The 4up button didn't exist before but it logged (and still logs) clicks to the user's initials to turn 4-up off.